### PR TITLE
Registers Editor: Implement PPU/SPU reservation control

### PR DESCRIPTION
**Pull request by elad335**
It might be super hacky (and not useful) to allow to change reservation address or acquire a reservation, BUT it is super useful to explicitly clear a reservation for debugging etc while it being not hacky.
Any value you write in the field for this "register" is valid and will cause a reservation lost, its name is "Reserv Invalidation".

### CODE REVIEW

File: rpcs3/rpcs3qt/register_editor_dialog.cpp [hunk 1]
```diff
@@ -125,6 +127,7 @@ register_editor_dialog::register_editor_dialog(QWidget *parent, u32 _pc, const s
 			m_register_combo->addItem("SRR0", +SPU_SRR0);
 		}
 
+		m_register_combo->addItem("Reserv Invalidation", +RESERVATION_LOST);
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/register_editor_dialog.cpp:130]: is this shortened intentionally?

Comment (elad335) [reply rpcs3/rpcs3qt/register_editor_dialog.cpp:130]: I think it would be ugly if its much longer than the other register names so yea.

Comment (Megamouse) [reply rpcs3/rpcs3qt/register_editor_dialog.cpp:130]: can't you at least put a dot on it?

Comment (elad335) [reply rpcs3/rpcs3qt/register_editor_dialog.cpp]: renamed.

**Pull request merged**